packages/vuetify/src/composables/layout.ts

```diff
@@ -206,11 +206,11 @@ export function createLayout (props: { overlaps?: string[], fullHeight?: boolean
     const layer = layers.value[layers.value.length - 1].layer
 
     return {
-      position: 'relative',
-      paddingLeft: convertToUnit(layer.left),
-      paddingRight: convertToUnit(layer.right),
-      paddingTop: convertToUnit(layer.top),
-      paddingBottom: convertToUnit(layer.bottom),
+      position: 'absolute',
+      left: convertToUnit(layer.left),
+      right: convertToUnit(layer.right),
+      top: convertToUnit(layer.top),
+      bottom: convertToUnit(layer.bottom),
```

Comment:
Is this necessary? It makes it so v-main has to be the scrolling element instead of body, and you can't have absolute drawers that scroll with the page.